tools/ci_build/github/apple/c/onnxruntime-mobile-c.podspec.template

```diff
@@ -8,12 +8,16 @@ Pod::Spec.new do |spec|
     spec.summary                = "ONNX Runtime Mobile C/C++ Pod"
     spec.platform               = :ios, "@IOS_DEPLOYMENT_TARGET@"
     spec.vendored_frameworks    = "onnxruntime.framework"
-    spec.weak_framework         = 'CoreML'
-    spec.source_files           = 'onnxruntime.framework/Headers/*.h'
+    spec.weak_framework         = [ @WEAK_FRAMEWORK@ ]
+    spec.source_files           = "onnxruntime.framework/Headers/*.h"
     spec.preserve_paths         = [ @LICENSE_FILE@ ]
     spec.description            = <<-DESC
     A pod for the ONNX Runtime Mobile C/C++ library.
                                     DESC
-    # TODO workaround - support arm64 iphonesimulator later
-    spec.user_target_xcconfig    = { "EXCLUDED_ARCHS[sdk=iphonesimulator*]" => "arm64" }
+    spec.library                 = "c++"
+    spec.user_target_xcconfig    = {
+        "OTHER_CPLUSPLUSFLAGS" => "-fvisibility=hidden -fvisibility-inlines-hidden",
```

Comment:
why do we need these flags?

Reply:
We are exposing internal functions now, these function has the visibility hidden compile settings, adding this to the project to avoid multiple xcode compile warnings